.github/workflows/auto-release.yml

```diff
@@ -11,16 +11,8 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
-      # Get PR from merged commit to master
-      - uses: actions-ecosystem/action-get-merged-pull-request@v1
-        id: get-merged-pull-request
+      - uses: cloudposse/github-action-auto-release@v1
         with:
-          github_token: ${{ secrets.REPO_ACCESS_TOKEN }}
-      # Drafts your next Release notes as Pull Requests are merged into "main"
-      - uses: release-drafter/release-drafter@v5
-        with:
-          publish: ${{ !contains(steps.get-merged-pull-request.outputs.labels, 'no-release') }}
           prerelease: false
+          publish: true
```

Comment:
```suggestion
          publish: false
```

Reply:
@osterman in previous version publish was true except `no-release` label

https://github.com/cloudposse/atmos/pull/528/files#diff-5a93af5afcdcd2bdc566f58652f8af7fec28b2d02acb9bec8ee97e9334362731L22

Reply:
Except, @aknysh disabled the entire workflow because it was problematic and probably didn't know about the feature flag.
<img width="1181" alt="image" src="https://github.com/cloudposse/atmos/assets/52489/e5d1f8f1-ab50-44ca-bf5b-8f028e45fbbe">